LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2020 Sequencing Analysis Support Core - Leiden University Medical Center
+Copyright (c) 2019 Leiden University Medical Center
```

Comment:
Is this year change correct?

Reply:
Very observant, it isn't. The repository did not exist in 2019. I will fix that.